src/iframe/src/colors.js

```diff
@@ -38,6 +38,19 @@ const colors = {
   },
   triplet (i) {
     return triplets[i % triplets.length]
+  },
+
+  // NOTE: if triplet isn't a color in the pallet, this will return undefined.
```

Comment:
Can this return null instead of undefined?

Reply:
This didn't get fixed before the PR was merged. Want me to make another PR making the change?